front/migrations/20241211_parents_migrator.ts

```diff
@@ -489,6 +511,13 @@ async function migrateDataSource({
       break;
     }
 
+    // If all documents have the same timestamp, we set nextId to the last id
```

Comment:
Shouldn't it be if the last two are the same ?

Reply:
No, if all docs have the same timestam, it mean we're going to deathlooping. Last 2 would be fine if at least some docs at the beginning have a different ts (means we're not going to do the exact same query)